TruckBooking/src/main/java/com/TruckBooking/TruckBooking/Controller/LoadController.java

```
@@ -11,53 +17,76 @@
 import org.springframework.web.bind.annotation.RestController;
 import com.TruckBooking.TruckBooking.Entities.Load;
 import com.TruckBooking.TruckBooking.Model.LoadRequest;
-import com.TruckBooking.TruckBooking.Response.CreateLoadResponse;
-import com.TruckBooking.TruckBooking.Response.DeleteLoadResponse;
-import com.TruckBooking.TruckBooking.Response.UpdateLoadResponse;
 import com.TruckBooking.TruckBooking.Service.LoadServiceImpl;
 
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
 @RestController
 public class LoadController {
 
 	@Autowired
 	public LoadServiceImpl loadService;
-	
+
 	@GetMapping("/home")
 	public String getmessage() {
 		return "Welcome to loadApi git actions second check...!!!";
 	}
-	
+
 	@PostMapping("/load")
-	public CreateLoadResponse load(@RequestBody LoadRequest loadrequest) {
-		return loadService.addLoad(loadrequest);
+	public ResponseEntity<Object> load(@Valid @RequestBody LoadRequest loadrequest) {
```

here we should throw an exception right.. 
Like throws EntityNotFound Exception etc.

---

I dont think we should throw an entitynotfound here.
it is thrown when we get the data.
In post we r only posting the data, so no need here.
In others( get,put,delete) we should add this).

---

ha in get, put.. need to add changes..

---

I had those like throws EntityNotFoundException in every controller get or put, but If you closely look into the code, the exception is thrown mostly from service class. So, I felt like they are not necessary in controller class. It is still working fine. Anyway, you can suggest any case where this might fail.

---

My saying is, when ever we throw an exception, we must take care of it.. either through try-catch or a parent method should inherit it..

---

and I am throwing exception objects from service class to be handled.